src/tools/ScenarioMeasurement/Startup/Startup.cs

```diff
@@ -25,6 +25,7 @@ enum MetricType
     WinUI,
     WinUIBlazor,
     TimeToMain2,
+    BuildTime,
```

Comment:
Unsure whether `BuildTime` qualifies as part of "Startup.cs" but seems like it is sharing a lot of the infrastructure here.

Could we add test for it to "StartupTests.cs"?

Reply:
Yes, we are able to reuse code and just implement a new parser.

I reviewed the tests and noticed that all artifact-related tests are currently disabled. We could add a binlog and write a test, but it would only serve as a one-off check -- since the input is artificial and parser is unlikely to change.

Let me know what you think.

Reply:
> parser is unlikely to change
Not sure how likely it is that we extend the parser to measure additional build part. However, I agree that we might not need the test, especially since we don't test some other parsers either (e.g., parsing ADB log to extract startup time).